project.clj

```diff
@@ -1,10 +1,12 @@
-(defproject helpshift/sensu-client "0.11.0"
+(defproject helpshift/sensu-client "0.12.0"
   :description "Library to send alerts to sensu-client."
   :url "https://www.helpshift.com"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [org.clojure/data.json "0.2.6"]]
+  :dependencies [[org.clojure/clojure "1.8.0"]
+                 [org.clojure/data.json "0.2.6"]
+                 [cheshire "5.10.0"]
```

Comment:
`org.clojure/data.json` is already there so cheshire is not required. 

But I think we should repalce data.json with cheshire as we are using cheshire in most of our other projects. I will make this change in a separate commit.